streamer/src/quic.rs

```diff
@@ -42,6 +41,8 @@ pub const MAX_UNSTAKED_CONNECTIONS: usize = 500;
 
 // This will be adjusted and parameterized in follow-on PRs.
 pub const DEFAULT_QUIC_ENDPOINTS: usize = 1;
+// inlined to avoid solana-sdk dep
```

Comment:
I don't understand why do we need to redefine it here if we import sdk for development anyways.